tests/test_dicom/test_header_tweaks.py

```diff
@@ -209,12 +214,75 @@ def test_electron_density_append():
         + [ORIGINAL_DICOM_FILENAME, ADJUSTED_DICOM_FILENAME]
         + excess_adjustment_map_flat)
 
-    subprocess.check_call(command)
+    compare_dicom_cli(command, original_dicom_file, expected_dicom_file)
 
-    cli_adjusted_dicom_file = pydicom.read_file(
-        ADJUSTED_DICOM_FILENAME, force=True)
 
-    assert str(expected_dicom_file) == str(cli_adjusted_dicom_file)
+def test_structure_name_parse():
+    structure_names = [
+        'a RED=1', 'b', 'c', 'd RED=2.2', 'e red = 3', 'f', 'g Red: 4.7',
+        'h  RED=0.5  '
+    ]
+    expected_adjustment_map = {
+        'a RED=1': 1,
+        'd RED=2.2': 2.2,
+        'e red = 3': 3,
+        'g Red: 4.7': 4.7,
+        'h  RED=0.5  ': 0.5
+    }
```

Comment:
@msobolewski I left the naming using regex, but I made it be more robust to odd naming choices.